src/nodes/EnvironmentalEffects/X3DFogNode.js

```diff
@@ -67,6 +67,30 @@ x3dom.registerNodeType(
              * @instance
              */
             this.addField_SFFloat( ctx, "visibilityRange", 0 );
+
+            /**
+             * FogHeight is in world scale like visibilityRange.
```

Comment:
`fogHeight is the maximum height of fog above 0 in world scale. A value of zero disables the effect.` Is that accurate ?

Reply:
I think it is correct.